test/user.test.js

```diff
@@ -205,6 +205,73 @@ describe('User', function() {
       });
     });
 
+    describe('Password validation with Maxlength', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
```

Comment:
pass73 is a good password with 73 chars, so the expected behaviour that it cannot be used to neither create nor reset, while badpass is a completely bas password that cannot be used to login after the user is already created.